R/tar_funcs.R

```diff
@@ -0,0 +1,846 @@
+#' these are utility functions to facilitate targets pipeline
```

Comment:
yeah thats a good idea. Going forward I think it makes sense to split up `target` specific functions and just use the `tar_` prefix in the file names (i.e `tar_performance`, `tar_chirps`). A naming syntax like this could help make it clear that these are not generic functions, but specific to this data and the pipeline